### PR TITLE
Export initializeServices for host services to update designer services

### PR DESCRIPTION
**Pull request by shuktika12163**
Currently, designer services are initialized once in BJSWorkflowProvider and stored globally. They are not re-initialized on re-mounting the component. Exporting the function for host services to re-initialize the designer services as per need.

### CODE REVIEW

**Review by edgarguoms — APPROVED** [review]

File: libs/designer/src/lib/core/index.ts [hunk 1]
```diff
@@ -9,3 +9,4 @@ export { useNodeDisplayName } from './state/workflow/workflowSelectors';
 export { serializeWorkflow } from './actions/bjsworkflow/serializer';
 export { clearPanel, switchToWorkflowParameters } from './state/panel/panelSlice';
 export { useSelectedNodeId } from './state/panel/panelSelectors';
+export { initializeServices } from './state/designerOptions/designerOptionsSlice';
```
Comment (wifun) [inline libs/designer/src/lib/core/index.ts:12]: this is just for convenience right? currently the host can import from `.../designerOptionsSlice` since it's already exported

Comment (ek68794998) [reply libs/designer/src/lib/core/index.ts:12]: Good point, it looks like this is actually available:

```ts
import { initializeServices } from '@microsoft/logic-apps-designer/lib/core/state/designerOptions/designerOptionsSlice';
```

Should we just do that?

Comment (shuktika12163) [reply libs/designer/src/lib/core/index.ts:12]: I remember it not getting recognized and there being build errors when I tried using it without this change. Let me try again.

Comment (shuktika12163) [reply libs/designer/src/lib/core/index.ts:12]: This is the error I get when I try to import it without the change in LAUX:

Module not found: Can't resolve '@microsoft/logic-apps-designer/lib/core/state/designerOptions/designerOptionsSlice' in 'D:\power-platform-ux\packages\powerautomate-core\src\designer\v3Designer'

Comment (shuktika12163) [reply libs/designer/src/lib/core/index.ts:12]: image

**Review by hartra344 — APPROVED** [review]

File: libs/designer/src/lib/core/index.ts (hunk 1 above)
Comment (hartra344) [reply libs/designer/src/lib/core/index.ts:12]: It's not for convenience. It tells rollup and our build processes that it needs to be made available when it's bundled.

Comment (wifun) [reply libs/designer/src/lib/core/index.ts:12]: Hm ok, so it seems like the typing shouldn't be generated then because the IDE doesn't complain

Comment (hartra344) [reply libs/designer/src/lib/core/index.ts:12]: I mean you can import types that way, it just wont work if you try to import js code that way. A lot of libraries export this way.
